explainer.md

```diff
@@ -1056,6 +1020,32 @@ Relaxing the length to 1024 characters makes it less likely that bucket names
 can be
 
 
+
+### Bucket titles
+
+Because buckets are expected to be named using programmer-friendly identifiers,
+simiarly to variable names, we could have a bucket title option,
+which in contrast would be user-friendly descriptions of the bucket.
+Titles could be useful for user agents that want to offer the ability to
+delete individual buckets in their storage management UI. These user agents could
+display bucket titles when showing buckets to their users.
+
+`title` was chosen for consistency with the
+[HTML title element](https://html.spec.whatwg.org/multipage/semantics.html#the-title-element).
+
+Bucket titles could present some subtleties for applications that support multiple
+languages. Specifically, a bucket's title could presumably reflect the
+users' preferred language at the time the bucket is created. In this alternative
+the API surface will not allow changing a bucket's description, so already-created
+buckets will not reflect language preference changes.
+
+This alternative was rejected because including strings provided by the author in
+user agent UI may introduce a11y and i18n issues, as well as have non-trivial
+security and privacy implications, which may deter some user agents from using
+the `title` as intended. For example, user agents that intend to incorporate
+the `title` need to mitigate against misleading values such as
+`"You have a virus! Go to www.evil.com for a cleanup"`.
+
 ### Alternative name for the bucket `title` property
```

Comment:
`title` isn't in the main proposal anymore, so this isn't a self-contained alternative anymore. It may be better suited as a sub-section of the previous alternative.